src/data/projects.yml

```diff
@@ -156,3 +156,42 @@ categories:
         displayLang: JavaScript
         searchLang: javascript
         stars: 11
+  - title: Native Development
+    url: https://github.com/Esri?q=topic%3Anative-development
+    anchor: native-development
+    color: yellow
+    projects:
+      - title: Indoor Routing Xamarin
```

Comment:
let's continue to iterate on it. i wanted to get something up to address the immediate concern. this will work into our plan #66 to curate more projects